mpas_analysis/ocean/sst_timeseries.py

```diff
@@ -88,17 +107,22 @@ def sst_timeseries(config, streamMap=None, variableMap=None):
 
     if ref_casename_v0 != 'None':
         print '  Load in SST for ACMEv0 case...'
+        inRefDate = '0001-01-01 00:00:00'
+
         infiles_v0data = '{}/SST.{}.year*.nc'.format(indir_v0data,
                                                      ref_casename_v0)
```

Comment:
The alternative to making the config file more verbose would be to have some automated way of getting this info from the previous model runs and observations.  I'm not very well versed with how either of these data sets is stored or what information is available.  I would think that maybe the solution here would be to preprocess both observations and previous runs to use *real* dates (if that has a meaning, which it obviously wouldn't for a climatology for example).